Some corrections and an additions (#15)
Human-Connection/Documentation: server-api/installation.md

```bash
$ sudo docker-compose up --build
```

> . You see no more changes in your terminal-window?

Reviewer: ```suggestion
. This may take a while, depending on your hardware and your internet connection.
```

> Congratulations: Your server is now running at [http://localhost:3030](http://localhost:3030) which you can load in your browser. Some contributions wait for you at [http://localhost:3030/contributions](http://localhost:3030/contributions).

Reviewer: ```suggestion
After a while, your server is running at [http://localhost:3030](http://localhost:3030) which you can load in your browser. Some contributions wait for you at [http://localhost:3030/contributions](http://localhost:3030/contributions).
```

### Setting up the database seeder for local development 

Change the configuration in `config/local-development.json` or `config/local.json` and rerun `docker-compose up --build`.

> If both files do no exist, create a `config/local.json` , copy the content of `config/local.example.json` into it and rerun `docker-compose up --build`.

Reviewer: ```suggestion
If both files do not exist, run `cp config/local.example.json config/local.json` and rerun `docker-compose up --build`.
You can learn more about the purpose of these configuration files and the order how they are loaded in the wiki of `node-config`:
```
